### ISSUE TITLE
What is the purpose of `sysconfdir` in rust installation?

### ISSUE THREAD

**Issue by alitariq4589** [rust-lang/rust#130270]
I am trying to build rust master branch from source on RISC-V architecture. On, execution of `./x.py install`, I get the following error:

```
Building bootstrap
   Compiling bootstrap v0.0.0 (/home/user/Downloads/rust-1.81.0/rust/rust/src/bootstrap)
    Finished `dev` profile [unoptimized] target(s) in 36.42s
WARNING: The `change-id` is missing in the `config.toml`. This means that you will not be able to track the major changes made to the bootstrap configurations.
NOTE: to silence this warning, add `change-id = 129473` at the top of `config.toml`
warning: unable to check if origin/master is old due to error: No such file or directory (os error 2)
warning: origin/master is used to determine if files have been modified
warning: if it is not updated, this may cause files to be needlessly reformatted
DESTDIR is set to: "/etc"
thread 'main' panicked at src/core/build_steps/install.rs:92:9:
User doesn't have write access on `install.sysconfdir` path in `config.toml`.
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Build completed unsuccessfully in 0:00:39
```

The sysconfdir is set to `/etc` and why does it require the write access? This kills the whole purpose of building from source (i.e. building in specific user space)

**Comment by alitariq4589** [comment]
@ChrisDenton Thanks for the response. 

Then I assume these configuration files can be installed in any location as the user desires and will not alter the installation process. 

I already modified the install.prefix but it didn't change sysconfdir, so it struck me as an anomaly that I tried installing the package in the location where root/sudo privileges are not required but it still asked me for elevated privileges.

I think, it should be mentioned in the documentation explicitly, that if you want to build and store the binaries in a user-specific location instead of the system default location, then `install.prefix` AND `install.sysconfdir` both be changed via the command line.

**Comment by workingjubilee** [comment]
Please PR any improvements to the documentation that you desire.

**Comment by iyernaveenr** [comment]
> Please PR any improvements to the documentation that you desire.

Please find the PR below:
https://github.com/rust-lang/rust/pull/131001

Thanks!

**Comment by jieyouxu** [comment]
Closing as PR is merged.
